packages/ui/src/components/PreviewItem/PreviewItem.test.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+import "@testing-library/jest-dom";
+import { ComponentPropsWithoutRef } from "react";
+
+import { axeTest, renderWithProviders } from "@gc-digital-talent/jest-helpers";
+
+import PreviewItem from "./PreviewItem";
+import { previewDetails } from "./PreviewItem.stories";
+
+const renderPreviewItem = (
+  props: ComponentPropsWithoutRef<typeof PreviewItem>,
+) => {
+  return renderWithProviders(<PreviewItem {...props} />);
+};
+
+const defaultProps = {
+  title: "Preview Item",
+  details: previewDetails,
+};
+
+describe("Tabs", () => {
```

Comment:
Copy pasta

Reply:
Fixed in [1b61361](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11192/commits/1b61361bb455f70862e28360b6cee8603050063e)